test/format/t.c

```diff
@@ -51,6 +64,16 @@ main(int argc, char *argv[])
 
 	(void)testutil_set_progname(argv);
 
+#ifdef SIGALRM
```

Comment:
These ifdefs are surprising enough that I think they warrant a comment. Something like "Windows and Linux support a different set of signals, so be conservative about installing handlers."

Reply:
Added in e9f44ce1b.